src/app/components/ErrorPage/index.jsx

```diff
@@ -50,38 +49,32 @@ const ErrorPage = ({
   script,
   service,
 }) => (
-  <main role="main">
```

Comment:
`Skip to Content` does not work when `main` is not defined, we need to retain this for accessibility purposes.